src/PlatformClient.ts

```diff
@@ -49,6 +50,10 @@ export class PlatformClient extends PlatformResources {
         }
     }
 
+    abortPendingGetRequests() {
```

Comment:
If I understand well, this is the method we must call to avoid a throw?

Reply:
Exactly, this is the method that should be called when you want to abort all pending get requests coming from the platform-client